core/jobs_test.py

```diff
@@ -363,54 +363,64 @@ def test_cancelling_one_unfinished_job(self):
         self.assertIsNone(DummyJobManager.get_error(job2_id))
 
     def test_compress_output_list_with_single_char_outputs(self):
-        sample_output_list = [1, 2, 3, 4, 5]
-        expected = ['1', '2', '3', '<TRUNCATED>']
-        actual = jobs.BaseJobManager._compress_output_list(  # pylint: disable=protected-access
-            sample_output_list, test_only_max_output_len=3)
-        self.assertEquals(expected, sorted(actual))
+        input_list = [1, 2, 3, 4, 5]
+        expected_output_possibilities = [
```

Comment:
I would advise against making this change. Tests are stronger if the expected output is more deterministic. Does the canonical order not work?